<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPasswordDialog</class>
 <widget class="QDialog" name="ResetPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="codeLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>560</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #FFD700, stop:1 #FF69B4); /* Gradient galben (#FFD700) spre roz (#FF69B4) */
    color: white; /* Text alb pentru contrast */
    border: 2px solid #FFFFFF; /* Bordură albă */
    border-radius: 10px; /* Colțuri rotunjite */
    padding: 8px; /* Spațiu interior */
    font-size: 14px; /* Mărime font (poate fi ajustată și din proprietatea font) */
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #FFEC8B, stop:1 #FF8C94); /* Gradient mai deschis la hover */
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #DAA520, stop:1 #FF1493); /* Gradient mai închis la apăsare */
}</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
